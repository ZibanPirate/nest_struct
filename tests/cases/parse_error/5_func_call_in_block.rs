#![allow(dead_code)]
#[macro_use]
extern crate nest_struct;

#[nest_struct]
struct Nested {
    id: u32,
    name: nest! {
        some_func_call("test", 123);
    },
}

fn main() {}
