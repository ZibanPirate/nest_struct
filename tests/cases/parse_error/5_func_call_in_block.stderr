error: only struct and enum are supported inside nest! block
 --> tests/cases/parse_error/5_func_call_in_block.rs:9:9
  |
9 |         some_func_call("test", 123);
  |         ^^^^^^^^^^^^^^
